QuizApp/QuizApp/UI/User/UserViewController.swift

```diff
@@ -100,6 +167,23 @@ extension UserViewController: ConstructViewsProtocol {
             $0.leading.trailing.equalToSuperview().inset(20)
         }
 
+        nameLabel.snp.makeConstraints {
+            $0.top.equalTo(usernameTextField.snp.bottom).offset(20)
+            $0.leading.trailing.equalToSuperview().inset(20)
+        }
+
+        nameTextField.snp.makeConstraints {
+            $0.top.equalTo(nameLabel.snp.bottom).offset(10)
+            $0.leading.equalToSuperview().inset(20)
+        }
+
+        saveButton.snp.makeConstraints {
+            $0.top.equalTo(nameLabel.snp.bottom).offset(10)
+            $0.trailing.equalToSuperview().inset(20)
```

Comment:
This combination of trailing, leading, and width constraints will potentially create a breaking constraint warning in the logs. To avoid that you can:
* remove the width constraint
* use $0.trailing.lessThanOrEqualToSuperview().inset(20)